### PR TITLE
[hi] Add content/hi/docs/tutorials/kubernetes-basics/update/update-intro.html

### PR DESCRIPTION
**Pull request by verma-kunal**
This PR is a part of ongoing `hi` localization effort here #29353

/language hi

### CODE REVIEW

File: content/hi/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+                    </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+
+                <p>एप्लिकेशन स्केलिंग के समान, यदि कोई डिप्लॉयमेंट सार्वजनिक रूप से एक्सपोज़ होता है, तो सर्विस केवल अपडेट के दौरान उपलब्ध पॉड्स पर ट्रैफ़िक को लोड-बैलेंस करेगी। एक उपलब्ध पॉड एक उदाहरण है जो एप्लिकेशन के उपयोगकर्ताओं के लिए उपलब्ध है|</p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/update/update-intro.html:99]:
```suggestion
                <p>एप्लिकेशन स्केलिंग के समान, यदि कोई डिप्लॉयमेंट सार्वजनिक रूप से एक्सपोज़ होता है, तो सर्विस केवल अपडेट के दौरान उपलब्ध पॉड्स पर ट्रैफ़िक को लोड-बैलेंस करेगी। उपलब्ध पॉड एक इंस्टेंस है जो एप्लिकेशन के उपयोगकर्ताओं के लिए उपलब्ध है।</p>
```

File: content/hi/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+                <p>रोलिंग अपडेट निम्नलिखित क्रियाओं की अनुमति देते हैं:</p>
+                <ul>
+                    <li>किसी एप्लिकेशन को एक परिवेश से दूसरे परिवेश में प्रचारित करें (कंटेनर इमेज अपडेट के माध्यम से)</li>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/update/update-intro.html:103]:
```suggestion
                    <li>किसी एप्लिकेशन को एक एनवायरनमेंट से दूसरे एनवायरनमेंट में प्रचारित करें (कंटेनर इमेज अपडेट के माध्यम से)</li>
```

File: content/hi/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                    <li>पिछले संस्करणों में रोलबैक करें</li>
+                    <li>शून्य डाउनटाइम के साथ एप्लीकेशन का निरंतर इंटीग्रेशन और निरंतर डिलीवरी</li>
+
+                </ul>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i>यदि कोई डिप्लॉयमेंट सार्वजनिक रूप से प्रकट होता है, तो सर्विस अपडेट  के दौरान केवल उपलब्ध पॉड्स पर ट्रैफ़िक लोड-बैलेंस करेगी|</i></p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/update/update-intro.html:112]:
```suggestion
                    <p><i>यदि कोई डिप्लॉयमेंट सार्वजनिक रूप से प्रकट होता है, तो सर्विस अपडेट के दौरान केवल उपलब्ध पॉड्स पर ट्रैफ़िक लोड-बैलेंस करेगी।</i></p>
```

File: content/hi/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+                </div>
+            </div>
+        </div>
+
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p>निम्नलिखित इंटरैक्टिव ट्यूटोरियल में, हम अपने एप्लिकेशन को एक नए संस्करण में अपडेट करेंगे, और रोलबैक भी करेंगे|</p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/update/update-intro.html:121]:
```suggestion
                <p>निम्नलिखित इंटरैक्टिव ट्यूटोरियल में हम अपने एप्लिकेशन को एक नए संस्करण में अपडेट और रोलबैक करेंगे। </p>
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Final Review

File: content/hi/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+          <h3>उद्देश्यों</h3>
+                <ul>
+                    <li>kubectl का उपयोग करके रोलिंग अपडेट करें|</li>
```
[the remark this answers is not in the document]
Comment (anubha-v-ardhan) [reply content/hi/docs/tutorials/kubernetes-basics/update/update-intro.html:24]: Note that `|` and `।` are different characters. While `|` is a vertical bar, `।` is what we call पूर्ण विराम in Hindi.

**Comment by verma-kunal** [comment]
Hey @anubha-v-ardhan , are these commits good to go for squashing?

**Comment by anubha-v-ardhan** [comment]
Just one suggestion is pending, try to commit it and its good for squashing.

**Comment by anubha-v-ardhan** [comment]
/lgtm
/approve

Thank you @verma-kunal

**Pull request merged**